plugins/lsp/language-servers/default.nix

```
@@ -158,6 +158,11 @@ with lib; let
       description = "Enable denols, for Deno";
       package = pkgs.deno;
     }
+    {
+      name = "dhall_lsp_server";
+      description = "Enable dhall-lsp-server, for Dhall.";
+      package = pkgs.dhall-lsp-server;
+    }
```

```suggestion
    {
      name = "dhall-lsp-server";
      description = "Enable dhall-lsp-server, for Dhall.";
      serverName = "dhall_lsp_server";
    }
```
We use the `foo-bar` naming convention for language servers.
Here, we then need to set `serverName` to the real server name (`dhall_lsp_server`).
The package name though is set by default to the server name.